mlir/utils/jenkins/Jenkinsfile

```diff
@@ -865,13 +866,12 @@ pipeline {
                             dir('build') {
                                 sh 'date --utc +%Y-%m-%d > perf-run-date'
                                 // Run MLIR perf benchmarks
-                                sh 'python3 ./bin/perfRunner.py'
```

Comment:
This run used the default conv configs.  Added those into the conv-configs file, and now we don't need this run, which avoids overwriting the implicit output file.